src/copier.py

```diff
@@ -78,7 +78,7 @@ def copy_file(self, file_info, overwrite, dryrun):
             org_url = file_info[FILE_PATH]
             file_name = file_info[FILE_NAME_DEFAULT]
             self.log.info(f'Processing {org_url}')
-            key = f'{self.prefix}/{file_name}'
+            key = f'{self.prefix}/{file_name}'.strip('/')
```

Comment:
I believe this will change the location of the files uploading to S3. S3 treat **/abc/file.txt** different from **abc/file.txt**. We need to maintain the leading "/" if it's in the prefix, otherwise the file locations won't be consistent with FE uploaded file. I believe you can use os.path.join function in line 101 to fix this issue.

Reply:
Hi ming, self.bucket.upload_file_obj(key, stream, t_config) will join bucket and key.  The key should not start with "/".  This is the reason I make this change.